source/_events.html.erb

```diff
@@ -1,102 +1,3 @@
-<aside>
-  <a href="http://www.meetup.com/Ruby-On-Rails-Oceania-Sydney/events/220739727/" target="blank">
-    <div class="event">
-      <div class="card">
-        <div class="front">
-         <div class="date">
-            <span class="day">
-              24
-            </span>
-            <span class="month">
-              March
-            </span>
-          </div>
-          <div class="title">
-            Installfest Sydney
-          </div>
-          <div class="location">
-            Sydney Fairfax Media
-          </div>
-          <span class="pipe">/</span>
-          <div class="availability">
-            <strong>
-              Book your spot now!
-            </strong>
-          </div>
-        </div>
-        <div class="back">
-          <i class="icon-chevron-right more"></i>
-          <p>RSVP on Meetup.com</p>
-        </div>
-      </div>
-    </div>
-  </a>
-</aside>
-<aside>
-  <a href="http://www.meetup.com/Ruby-On-Rails-Oceania-Melbourne/events/220633428/" target="blank">
-    <div class="event">
-      <div class="card">
-        <div class="front">
-         <div class="date">
-            <span class="day">
-              16
-            </span>
-            <span class="month">
-              April
-            </span>
-          </div>
-          <div class="title">
-            Installfest Melbourne
-          </div>
-          <div class="location">
-           Envato, Melbourne
-          </div>
-          <!-- <span class="pipe">/</span> -->
-          <div class="availability">
-            <strong>
-              Book your spot now! <!-- full -->
-            </strong>
-          </div>
-        </div>
-        <div class="back">
-          <i class="icon-chevron-right more"></i>
-          <p>RSVP on Meetup.com</p>
-        </div>
-      </div>
-    </div>
-  </a>
-</aside>
-<aside>
-  <a href="http://www.meetup.com/BrisRuby/events/219951074/" target="blank">
-    <div class="event">
-      <div class="card">
-        <div class="front">
-         <div class="date">
-            <span class="day">
-              17
-            </span>
-            <span class="month">
-              March
-            </span>
-          </div>
-          <div class="title">
-            Installfest Brisbane
-          </div>
-          <div class="location">
-            River City Labs, Fortitude Valley
-          </div>
-          <span class="pipe">/</span>
-          <div class="availability">
-            <strong>
-              Book your spot now! <!-- full -->
-            </strong>
-          </div>
-        </div>
-        <div class="back">
-          <i class="icon-chevron-right more"></i>
-          <p>RSVP on Meetup.com</p>
-        </div>
-      </div>
-    </div>
-  </a>
-</aside>
\ No newline at end of file
+<%= partial :event_card, locals: { url: 'http://www.meetup.com/BrisRuby/events/219951074/', day: 17, month: 'March', title: 'Installfest Brisbane', location: 'River City Labs' } %>
```

Comment:
is dropping suburb on the location intentional? possibly worth making it consistent across all events